src/backend/types/internal/metadata.ts

```diff
@@ -3,29 +3,29 @@
 export module v1Types {
     export interface AnalyzerStats {
         version: string;
-    
+
```

Comment:
There is a jslint rule to avoid these issues: https://eslint.org/docs/rules/no-trailing-spaces. Please enable it in this file: https://github.com/Ericsson/CodecheckerVSCodePlugin/blob/main/.eslintrc.json. So we can avoid these problems next time.

Reply:
I'll add this rule (along with a couple of others) in a separate PR, after this one is merged.

Reply:
Okay, let's do it in a separate PR but I have created an issue for it (#11).